src/app/not-found.js

```diff
@@ -1,5 +1,18 @@
 const NotFound = () => {
```

Comment:
you need to use flex instead of paddings because otherwise it won't take up the full page (notice how footer is not touching the bottom of the page like it normally should)


delete text-center and all "p" styles and try to get the same visual with flex
https://acm-ucr.github.io/docusaurus/docs/tailwind/flexbox

you can get the flex container to stretch to the full page using flex-1
https://tailwindcss.com/docs/flex